File: Euclid/Messages/Outgoing/WALLETBALANCE.cs
using Euclid.Network.Streams.Util;

namespace Euclid.Messages.Outgoing
{
    internal class WALLETBALANCE : IMessageComposer
    {
        private int credits;

        public WALLETBALANCE(int credits)
        {
            this.credits = credits;
        }

        public override void Write()
        {
            this.Data.Add(new ArgumentEntry(credits));
        }
    }
}

File: euclid/src/main/java/euclid/messages/outgoing/WALLETBALANCE.java
package euclid.messages.outgoing;

import euclid.messages.IMessageComposer;
import euclid.network.streams.util.ArgumentEntry;

public class WALLETBALANCE extends IMessageComposer {

    private final int credits;

    public WALLETBALANCE(int credits) {
        this.credits = credits;
    }

    @Override
    public void write() {
        getData().add(new ArgumentEntry(credits));
    }
}
